schemas/vax.schema

```diff
@@ -5,27 +5,30 @@ DROP SCHEMA IF EXISTS $PG_SCHEMA_VAX CASCADE;
 CREATE SCHEMA $PG_SCHEMA_VAX;
 
 CREATE TABLE $PG_SCHEMA_VAX.age (
-id   integer PRIMARY KEY,
-name varchar(28) NOT NULL
+id            INTEGER PRIMARY KEY,
+name          VARCHAR(28) NOT NULL,
+age_0         DECIMAL(3, 1) NOT NULL, 
+age_1         DECIMAL(3, 1), 
+is_high_risk  BOOLEAN NOT NULL
 );
 
 CREATE TABLE $PG_SCHEMA_VAX.race (
-id   integer PRIMARY KEY,
-name varchar(37) NOT NULL
+id   INTEGER PRIMARY KEY,
+name VARCHAR(37) NOT NULL
 );
```

Comment:
Hey @brandomr, I've been trying to figure out race and ethnicity.  I was going for a pair of tables in the `main` schema.  What I've came up with so far (after sieving through recommendations from various organizations) is:

```
CREATE TABLE $PG_SCHEMA_MAIN.race (
id    integer PRIMARY KEY,
name  text
);

INSERT INTO $PG_SCHEMA_MAIN.race (id, name) VALUES (1, 'American Indian or Alaska Native');
INSERT INTO $PG_SCHEMA_MAIN.race (id, name) VALUES (2, 'Asian');
INSERT INTO $PG_SCHEMA_MAIN.race (id, name) VALUES (3, 'Black or African American');
INSERT INTO $PG_SCHEMA_MAIN.race (id, name) VALUES (4, 'Native Hawaiian or Pacific Islander');
INSERT INTO $PG_SCHEMA_MAIN.race (id, name) VALUES (5, 'White');
INSERT INTO $PG_SCHEMA_MAIN.race (id, name) VALUES (6, 'Other');
INSERT INTO $PG_SCHEMA_MAIN.race (id, name) VALUES (7, 'Multiple');

CREATE TABLE $PG_SCHEMA_MAIN.ethnic (
id    integer PRIMARY KEY,
name  text
);

INSERT INTO $PG_SCHEMA_MAIN.ethnic (id, name) VALUES (1, 'Hispanic/Latino');
INSERT INTO $PG_SCHEMA_MAIN.ethnic (id, name) VALUES (2, 'Non Hispanic/Latino');
```

It's a bit weird CDC would confound race and ethnicity the way they do, which is:

`Black only, non-Hispanic`, `Hispanic`, `Other or multiple races, non-Hispanic`, `White only, non-Hispanic`

I'm not sure how to disentangle the two to use central tables.  Not even sure if it's possible.  In dealing with the CDC Web site, have you came across any pointers to how we could do that?

Reply:
I agree this is odd. I feel like this is a particularly tricky area but they may be just sticking with conventions they established in 2010 to keep the longitudinal nature of the data even if the general approach to handling race/ethnicity in survey methodology may have changed. I'll see if they have more substantial descriptions around this.

Reply:
Thanks, Brandon. Good point. The more I think about this the more I'm convinced we will need several encodings of race/ethnicity and create a mapping between them.